src/exports.jl

```diff
@@ -83,9 +83,15 @@ export
     nfaces,
     addnodeset!,
     addfaceset!,
+    addboundaryfaceset!,
     addedgeset!,
+    addboundaryedgeset!,
     addvertexset!,
+    addboundaryvertexset!,
     addcellset!,
+    getfaceedges,
+    getfacevertices,
+    getedgevertices,
```

Comment:
These should be internals for now and not part of the public API.

After merging the current master you will see a new section in the documentation of internal APIs. We should move the functions there (section `Elements`).